documentation/modules/con-accessing-kafka-bridge-from-outside.adoc

```diff
@@ -0,0 +1,30 @@
+// This assembly is included in the following assemblies:
+//
+// assembly-kafka-bridge-overview.adoc
+
+[id='con-accessing-kafka-bridge-from-outside-{context}']
+
+= Accessing the Kafka Bridge outside of Kubernetes
```

Comment:
I think this is specific to the Stirmzi operators and should probably stay there.

Reply:
This we can add to the _deploying_ guide like _[Exposing the Kafka Connect API](https://strimzi.io/docs/operators/in-development/deploying.html#con-exposing-kafka-connect-api-str)_

Reply:
removed